Sources/ConsoleKit/Command/CommandGroup.swift

```diff
@@ -27,9 +27,9 @@ extension CommandGroup {
 extension CommandGroup {
     public func run(using context: inout CommandContext) throws {
         if let command = try self.commmand(using: &context) {
-            try command.run(using: &context)
+          try command.run(using: &context)
         } else if let `default` = self.defaultCommand {
-            return try `default`.run(using: &context)
+          return try `default`.run(using: &context)
```

Comment:
The repo is 4 spaces
```suggestion
            return try `default`.run(using: &context)
```